src/System.Device.Gpio.Tests/UnixDriverTests.Linux.cs

```diff
@@ -8,8 +8,8 @@ namespace System.Device.Gpio.Tests
 {
     public class UnixDriverTests : GpioControllerTestBase
     {
-        protected override GpioDriver GetTestDriver() => new UnixDriver();
+        protected override GpioDriver GetTestDriver() => UnixDriver.InitUnixDriver();
```

Comment:
Is this existing name? we usually have static Create method

Reply:
Sorry didn't understood what you mean by existin name?  the method signature is:
`public static UnixDriver InitUnixDriver() ` i think i named it by one of reviewers suggestion, do you mean it should be named like CreateUnixDriver?